src/cpp/shared/ITest.h

```diff
@@ -1,6 +1,11 @@
 #pragma once 
 
-#include "defs.h"
+#if __GNUC__ >= 4
```

Comment:
This will work incorrectly when compiler is neither cl nor gcc v4+, as it will resolve to `__declspec` which is cl-specific.